mdoc/src/main/scala/mdoc/internal/markdown/MarkdownFile.scala

```diff
@@ -20,12 +21,26 @@ final case class MarkdownFile(input: Input, file: InputFile, parts: List[Markdow
   }
 }
 object MarkdownFile {
+  object syntax {
+    private[markdown] implicit class StringOps(private val x: String) extends AnyVal {
+      def isNL: Boolean = x.forall { c => c == '\n' || c == '\r' }
```

Comment:
can't be named `isEmpty` as this name is reserved by `String.isEmpty` standard method